config/helm/chart/default/templates/_helpers.tpl

```diff
@@ -29,6 +29,8 @@ Check if default image or imageref is used
 {{- else -}}
     {{- if (.Values.imageRef).repository -}}
         {{- .Values.imageRef.tag | default (printf "v%s" .Chart.AppVersion) | printf "%s:%s" .Values.imageRef.repository -}}
+    {{- else if eq (include "dynatrace-operator.platform" .) "openshift" -}}
```

Comment:
how does it work with https://github.com/Dynatrace/dynatrace-operator/pull/3628?

Reply:
platform still exists -> https://github.com/Dynatrace/dynatrace-operator/pull/3628/files#diff-0425575c0ee7e3b71e1b1457ed08cb886f1d5d7526a474dc9388ff1b9de67fc7L16

Reply:
and here https://github.com/Dynatrace/dynatrace-operator/pull/3628/files#diff-096ff34b076d51e9cedae8837f8d85cf95173899d1f3e269c2d497e72b69cd03L18